src/PrizeVault.sol

```diff
@@ -538,25 +583,21 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
             _yieldFee = (_amountOut * FEE_PRECISION) / (FEE_PRECISION - _yieldFeePercentage) - _amountOut;
```

Comment:
This is before your time, but I think it would be good to add comments explaining how the yield fee is *on top* of the amount being requested.  The auditors didn't understand why it was being comptued this way; it merits explanation.

Reply:
Clarified in this commit: [cc89a26](https://github.com/GenerationSoftware/pt-v5-vault/pull/75/commits/cc89a26ed9673593975c8e29fce1ebeb8f5a2e3e)